luma/lcd/device.py

```diff
@@ -292,6 +292,95 @@ def contrast(self, value):
         self.command(0x21, 0x14, value | 0x80, 0x20)
 
 
+class st7789(backlit_device):
+    """
+    Serial interface to a colour ST7789 240x240 pixel LCD display.
+    """
```

Comment:
can you add `@since 2.9.0` here

Reply:
What would be the correct syntax for this decorator? `@since("2.9.0")`?

Reply:
See https://github.com/rm-hull/luma.core/blob/255fadf0e0699369966f02aad92f69f877325472/luma/core/util.py#L118 for an example